gist/jbuild

```diff
@@ -2,7 +2,7 @@
 
 (executables
  ((libraries (cohttp-lwt-unix github_unix cmdliner lambda-term))
-  (flags (:standard -principal -strict-sequence -g
+  (flags (:standard -principal -strict-sequence -g -safe-string
                     -warn-error "+1..47-D+49" -w "A-E-41-42-44-48"))
```

Comment:
can you remove the `-warn-error` flag as well while you are fixing this kind of stuff. This is not a good idea to let this in released software.